content/10.methods.md

```diff
@@ -86,9 +86,12 @@ To not give undue influence to papers with multiple corresponding authors, subse
 
 ### Estimation of Gender
 
-We predicted the gender of honorees and authors using the <https://genderize.io> API that produces predictions trained on over 100 million name-gender pairings collected from the web.
+We predicted the gender of honorees and authors using the <https://genderize.io> API, which produces predictions trained on over 100 million name-gender pairings collected from the web.
 We used author and honoree first names to retrieve predictions from genderize.io.
-The predictions, which consider gender as a binary trait, represent the probability of an honoree or author being male or female.
+The predictions represent the probability of an honoree or author being male or female.
+We use the estimated probabilities and do not convert to a hard group assignment
+For example, a query to <https://genderize.io> on January 26, 2020 for "Casey" returns a probability of male of .74 and a probability of female of 0.26, which we would add for an author with this first name.
+Because of the limitations of considering gender as a binary trait and inferring it from first names, we only consider predictions in aggregate and not as meaningful values for specific scientists.
```

Comment:
```suggestion
Because of the limitations of considering gender as a binary trait and inferring it from first names, we only consider predictions in aggregate and not as individual values for specific scientists.
```